diff --git a/src/components/FormEmbed.tsx b/src/components/FormEmbed.tsx
index 46858e336d19608e561366864b89afb52201208a..c16c368a356390c7429b604c61eb2dcb06defd6d 100644
--- a/src/components/FormEmbed.tsx
+++ b/src/components/FormEmbed.tsx
@@ -109,34 +109,39 @@ const FormEmbed: React.FC<FormEmbedProps> = ({ link, userData, teamData, matters
                     isMultiline={false}
                     onDismiss={() => setSubmissionSuccess(null)}
                     dismissButtonAriaLabel="Close"
                     styles={{ root: { marginBottom: '10px', borderRadius: '4px' } }}
                 >
                     {submissionSuccess}
                 </MessageBar>
             )}
             {link.embedScript ? (
                 <div ref={formContainerRef}>
                     {!isCognitoLoaded && (
                         <div style={loaderStyle}>
                             <img src={loaderIcon} alt="Loading..." style={{ width: '100px', height: 'auto' }} />
                         </div>
                     )}
                 </div>
             ) : link.fields ? (
                 <BespokeForm
                     key={formKey}
                     fields={link.fields.map((f) => ({ ...f, name: f.label }))}
                     onSubmit={handleFinancialSubmit}
                     onCancel={() => { }}
                     isSubmitting={isSubmitting}
                     matters={matters}
                 />
-                
+            ) : link.component ? (
+                React.createElement(link.component, {
+                    team: teamData || [],
+                    matters,
+                    onBack: () => {}
+                })
             ) : (
                 <div>No form available for this item.</div>
             )}
         </div>
     );
 };
 
-export default FormEmbed;
+export default FormEmbed;
