diff --git a/src/tabs/instructions/Instructions.tsx b/src/tabs/instructions/Instructions.tsx
index df7376df17b67883022fc7c36d955f28b7210bc6..0e0fc5842cccf4b3756c005db0ebbc984f0c2bfd 100644
--- a/src/tabs/instructions/Instructions.tsx
+++ b/src/tabs/instructions/Instructions.tsx
@@ -193,51 +193,51 @@ const Instructions: React.FC<InstructionsProps> = ({
                   key={opt.label}
                   title={opt.label}
                   icon={opt.icon}
                   isDarkMode={isDarkMode}
                   onClick={() => setNewMatterClientType(opt.label)}
                   selected={newMatterClientType === opt.label}
                   style={{ "--card-index": idx } as React.CSSProperties}
                 />
               ))}
           </div>
         ) : (
           <>
             <div className={quickLinksStyle(isDarkMode)}>
               <QuickActionsCard
                 title="New Matter"
                 icon="Calendar"
                 isDarkMode={isDarkMode}
                 onClick={() => {
                   setSelectedInstruction(null);
                     setNewMatterClientType("Individual");
                     setShowNewMatterPage(true);
                   }}
                   style={{ "--card-index": 0 } as React.CSSProperties}
                 />
                 <QuickActionsCard
-                  title="EID Check"
+                  title="Verify an ID"
                   icon="IdCheck"
                   isDarkMode={isDarkMode}
                   onClick={() => {
                     setSelectedInstruction(null);
                     setShowEIDPage(true);
                   }}
                   style={{ "--card-index": 1 } as React.CSSProperties}
                 />
                 <QuickActionsCard
                   title="Risk Assessment"
                   icon="Assessment"
                   isDarkMode={isDarkMode}
                   onClick={() => { }}
                   style={{ "--card-index": 2 } as React.CSSProperties}
                 />
                 <QuickActionsCard
                   title="Draft CCL"
                   icon="OpenFile"
                   isDarkMode={isDarkMode}
                   onClick={() => setShowRiskPage(true)}
                   style={{ "--card-index": 3 } as React.CSSProperties}
                 />
               </div>
               <div className={pivotBarStyle(isDarkMode)}>
                 <Pivot
