diff --git a/src/tabs/instructions/DocumentsV3.tsx b/src/tabs/instructions/DocumentsV3.tsx
index 7a880fbecace9cd85a20d2639d92b8869bf7ded1..fe957763017084c9ae95e87fd73ed169c2266401 100644
--- a/src/tabs/instructions/DocumentsV3.tsx
+++ b/src/tabs/instructions/DocumentsV3.tsx
@@ -2361,51 +2361,51 @@ Disbursement | Amount | VAT chargeable
                 before: 'Please provide us with £',
                 after: 'on account of costs'
             },
             'matter_number': {
                 before: 'Please use the reference <<',
                 after: '>> We work with money'
             },
             'and_or_intervals_eg_every_three_months': {
                 before: 'We have agreed to provide you with an update on the amount of costs when appropriate as the matter progresses',
                 after: '. Risk analysis'
             },
             'contact_details_for_marketing_opt_out': {
                 before: '—contacting us at',
                 after: '; —using the \'unsubscribe\' link'
             },
             'link_to_preference_centre': {
                 before: '—updating your marketing preferences on our',
                 after: '. Prevention of money laundering'
             },
             'may_will': {
                 before: 'Your matter',
                 after: 'involve court proceedings'
             },
             'instructions_link': {
                 before: 'The \'Instructions for Cancellation\' notice at',
-                after: 'explains: —how to cancel'
+                after: 'explains:'
             },
             'insert_next_step_you_would_like_client_to_take': {
                 before: '☐',
                 after: '[State why this step is important]'
             },
             'state_why_this_step_is_important': {
                 before: '[Insert next step you would like client to take]',
                 after: '☐ Provide a payment on account'
             },
             'state_amount': {
                 before: '☐ Provide a payment on account of costs and disbursements of £',
                 after: 'If we do not receive a payment'
             },
             'describe_first_document_or_information_you_need_from_your_client': {
                 before: '☐ Provide the following documents [and information] to allow me to take the next steps in your matter:',
                 after: '[describe second document]'
             },
             'describe_second_document_or_information_you_need_from_your_client': {
                 before: '[describe first document or information you need from your client]',
                 after: '[describe third document]'
             },
             'describe_third_document_or_information_you_need_from_your_client': {
                 before: '[describe second document or information you need from your client]',
                 after: 'Without these documents there may be a delay'
             },
